src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -237,12 +265,13 @@ export const ChannelView: React.FC = () => {
     <ViewWrapper>
       <ChannelCover assetUrl={coverPhotoUrl} />
       <LimitedWidthContainer>
+        {smMatch ? <CollectorsBoxContainer>{collectorBoxNode}</CollectorsBoxContainer> : null}
```

Comment:
```suggestion
```
Let's just delete this. Also, CollectorsBoxContainer doesn't work well, If you remove `CollectorsBoxContainer`(If I'm not mistaken, some channels will not have collectorsBox), the layout will be broken.
Look: 
![image](https://user-images.githubusercontent.com/51168865/150755446-af1b0603-c29b-4364-be8f-e38a8dc173fa.png)